go-apps/meep-rnis/sbi/rnis-sbi.go

```diff
@@ -235,5 +254,25 @@ func processActiveScenarioUpdate() {
 			}
 		}
 	}
+}
 
+func processGisEngineUpdate(assetMap map[string]string) {
+	for assetName, assetType := range assetMap {
+		// Only process UE updates
+		if assetType == postgis.TypeUe {
+			if assetName == postgis.AllAssets {
+				ueMap, err := sbi.pc.GetAllUe()
+				if err == nil {
+					for _, ue := range ueMap {
+						log.Debug("UE[", ue.Name, "] POA [", ue.Poa, "] distance[", ue.PoaDistance, "]")
```

Comment:
currently RNIS not storing it anywhere, only a log is there while waiting for Time Advance, valid approach, just add a comment please